dmc/customers/use_cases/create_webhook.rb

```diff
@@ -0,0 +1,10 @@
+module Customers
+  class CreateWebhook < Struct.new(:user, :remote_project)
+
+    def run!
+      account = GetAccount.new('github', user.id).run!
```

Comment:
How do you know it's GitHub? i agree, for the moment, we only have GitHub but do you think we should hardcode for GitHub?